classes/PodsInit.php

```diff
@@ -1863,6 +1866,38 @@ public function admin_bar_links() {
 
 	}
 
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$additional_profile_data" missing

<sub>hash: b836cf5802aeeab8f3fe5f98ec824207</sub>